src/main/java/com/sss/garage/model/track/Track.java

```diff
@@ -0,0 +1,54 @@
+package com.sss.garage.model.track;
+
+import com.sss.garage.model.event.Event;
+import jakarta.persistence.*;
+
+import java.util.Set;
+
+@Entity
+public class Track {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+
+    private String country;
```

Comment:
Jeśli chcemy do tego dorzucić jakieś flagi krajów itp to można z tego zrobić osobny model. A jak nie to nie

Reply:
Zależy w sumie od tego, jak to front widzi, dla mnie bez znaczenia